### ISSUE TITLE
Add ability to query based on CN

### ISSUE THREAD

**Issue by mpars0ns** [mpars0ns/maltego_censys#1]
After talking with a few people it was suggested to add a certificate CN entity and add the ability to do searches based on that CN.

**Comment by krmaxwell** [comment]
So this already exists, it looks like?

**Comment by mpars0ns** [comment]
Yup it does..but i need to retool it a bit because of some issues I have seen with it
